demos/dqx_demo_library.py

```diff
@@ -233,6 +233,20 @@
       column: col1
       aggr_type: count
       limit: 1.2
+- criticality: warn
```

Comment:
We don't necessarily have to update the demo. We have an example of aggr check already in there.